### ISSUE TITLE
error when downloading a file whose name contains ___

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#155]
```
What steps will reproduce the problem?
1. Upload file whose name contains ___
2. On client I see > instead ___
3.

What version of the product are you using? On what operating system?
Ubuntu 12.04. Gwt-upload 0.6.4



Please provide any additional information below.

package gwtupload.server;
...
public class UploadServlet extends HttpServlet implements Servlet {
...
protected static void renderXmlResponse(HttpServletRequest request, 
HttpServletResponse response, String message, boolean post) throws IOException {
...
    if (post) {
      xml = TAG_MSG_START + xml.replaceAll("<", TAG_MSG_LT).replaceAll(">", TAG_MSG_GT) + TAG_MSG_END;
    }
```

Original issue reported on code.google.com by `Demetriu...@gmail.com` on 10 Jul 2012 at 10:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 3 Dec 2012 at 7:42
- Changed state: **Accepted**
- Added labels: **Milestone-0.6.6**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision a22d3b898052.
```

Original comment by `manuel.carrasco.m` on 3 Dec 2012 at 7:46
- Changed state: **Fixed**
